### PR TITLE
Fix bug travis CI .

### PR DESCRIPTION
**Pull request by hardik124**
Added android emulator in travis CI.
Removed jdk 7 build .
Fix permission for Tools v 25 .
Fixes issue :  #60

### CODE REVIEW

**Comment by hardik124** [comment]
@kukreja-vikramaditya  , Please review .

File: .travis.yml [hunk 1]
```diff
@@ -40,6 +61,9 @@ cache:
 
 script:
```
Comment (kukreja-vikramaditya) [inline .travis.yml]: remove lines 66, 67
>     echo "Travis branch is $TRAVIS_BRANCH"
>     echo "Travis branch is in pull request $TRAVIS_PULL+REQUEST"

File: .travis.yml [hunk 2]
```diff
@@ -1,29 +1,50 @@
...
+    - ${ANDROID_HOME}/tools/emulator -avd test -no-skin -no-audio -no-window -netspeed full -netdelay none &
+    - ls -la
+    - cat /usr/local/bin/android-wait-for-emulator
+    - android-wait-for-emulator
+    - adb shell input keyevent 82 &
+
+after_failure:
+    - echo "Build failure"
+
+after_success:
```
Comment (kukreja-vikramaditya) [inline .travis.yml]: Remove after_script because we are not using Codecov

**Comment by kukreja-vikramaditya** [comment]
Rebase and squash all your commits into one.

**Pull request closed without merging**